frameworks/elastic/src/main/dist/svc.yml

```diff
@@ -281,6 +281,49 @@ pods:
           - name: node
             type: TLS
         {{/TASKCFG_ALL_SECURITY_ENABLED}}
+  exporter:
+    count: 1
+    {{#ENABLE_VIRTUAL_NETWORK}}
+      networks:
+        {{VIRTUAL_NETWORK_NAME}}:
+          labels: {{VIRTUAL_NETWORK_PLUGIN_LABELS}}
+    {{/ENABLE_VIRTUAL_NETWORK}}
+    uris:
+      - {{ELASTICSEARCH_JAVA_URI}}
+      - {{PROMETHEUS_EXPORTER_URI}}
+    tasks:
+      node:
+        goal: RUNNING
+        cpus: {{PROMETHEUS_EXPORTER_CPUS}}
+        memory: {{PROMETHEUS_EXPORTER_MEM}}
+        labels: "DCOS_METRICS_FORMAT:prometheus,DCOS_METRICS_PORT_INDEX:0,DCOS_METRICS_ENDPOINT:/metrics"
+        ports:
+          exporter:
+            port: 9114
+        volume:
+          path: "container-path"
+          type: {{PROMETHEUS_EXPORTER_DISK_TYPE}}
+          size: {{PROMETHEUS_EXPORTER_DISK}}
+          {{#PROMETHEUS_EXPORTER_VOLUME_PROFILE}}
+            profiles:
+              - {{PROMETHEUS_EXPORTER_VOLUME_PROFILE}}
+          {{/PROMETHEUS_EXPORTER_VOLUME_PROFILE}}
+        cmd: |
+          set -ex
+          export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/)
+          exec ./elasticsearch_exporter-1.1.0.linux-amd64/elasticsearch_exporter \
```

Comment:
Would it be better if the exporter directory (currently hardcoded with the `1.1.0` version) used instead the value from `ELASTICSEARCH_PROMETHEUS_EXPORTER_VERSION` or calculated like `JAVA_HOME` above?